README.md

```diff
@@ -18,6 +18,8 @@ Built with [hugo][1], a static website generation tool.
 
 6. Download the documentation repo `git clone https://github.com/DataDog/documentation.git`
 
+7. Run `yarn` in the documentation repo
+
```

Comment:
I don't think we should add this step as typically the make file will do this for you and will use `yarn install --frozen-lockfile` to ensure its using the frozen versions.

Reply:
Maybe we can strongly recommend pyenv further up? instead of just the callout. Also we could add or link to some hugo module troubleshooting. What do you think?

Reply:
yes I agree with you.
Emphasising a bit more on pyenv would be a good solution. 
Link to hugo module troubleshooting too

Reply:
@davidejones the thing is the corp-hugo repo is private whereas the Documentation repo is public, so adding the link to the troubleshooting wiki does not work 🧐